src/components/raycaster.js

```diff
@@ -54,12 +54,21 @@ module.exports.Component = registerComponent('raycaster', {
   update: function () {
     var data = this.data;
     var raycaster = this.raycaster;
+    var self = this;
 
     // Set raycaster properties.
     raycaster.far = data.far;
     raycaster.near = data.near;
 
     this.refreshObjects();
+
+    // Pick up any `Object3D`s that get set later (e.g., after model loading).
+    this.objects.forEach(object => {
+      if (!object.el) { return; }
+      object.el.addEventListener('object3dset', function (evt) {
+        self.objects.push(evt.detail.object);
```

Comment:
This is only correct to do if the object's element still satisfies the gathering criteria by the time the model finishes loading, right?  for example, the object has not changed to a non-selectable css class while the object was loading